flow/Makefile

```diff
@@ -89,6 +89,27 @@ DESIGN_CONFIG ?= ./designs/nangate45/gcd/config.mk
 # in this file. This allows the DESIGN_CONFIG to set different defaults than
 # this file.
 include $(DESIGN_CONFIG)
+export DESIGN_NICKNAME?=$(DESIGN_NAME)
+# default value "base" is duplicated from variables.yaml because we need it
+# earlier in the flow for BLOCKS. BLOCKS is a feature specific to Makefile
+# that will not be ported to bazel-orfs.
+export FLOW_VARIANT?=base
```

Comment:
Shouldn't this come from variables.yaml where the default is already specified?

Reply:
Yes, but it is an earth cooling problem. BLOCKS needs to be set up as part of make DESIGN_CONFIG. I think this is the lesser evil.

Reply:
Why do "BLOCKS needs to be set up as part of make DESIGN_CONFIG"?

Reply:
Because scripts later will do all sorts of messing about with wrap and libs and lefs and the bash script to set up variables need to run before make rules are set up

Reply:
Should the flow default values be setup before we even get here?

Reply:
no. Default values have priorities: first user, then platform, then variables.yaml